src/vs/editor/contrib/find/simpleFindWidget.ts

```diff
@@ -220,16 +221,20 @@ export abstract class SimpleFindWidget extends Widget {
 
 		setTimeout(() => {
 			dom.addClass(this._innerDomNode, 'visible');
+			dom.addClass(this._innerDomNode, 'visible-transition');
 			this._innerDomNode.setAttribute('aria-hidden', 'false');
 		}, 0);
 	}
 
 	public hide(): void {
 		if (this._isVisible) {
 			this._isVisible = false;
-
-			dom.removeClass(this._innerDomNode, 'visible');
+			dom.removeClass(this._innerDomNode, 'visible-transition');
 			this._innerDomNode.setAttribute('aria-hidden', 'true');
+			// Need to delay toggling visibility until after Transition, then visibility hidden - removes from tabIndex list
+			setTimeout(() => {
```

Comment:
Would it matter if the widget is toggled or shown again before the timeout has fired?  Or will this code still do the correct thing?

Reply:
@mjbvz 
Good point.  I based my tweak on the similar setTimeout in reveal() for settings focused. 200ms is right at the edge of human reaction , I certainly can't do anything using my Dragon voice dictation.  That said, it still could be a window.  I slowed down the transition for experimentation.  I think I closed one, probably the only window by not toggling _isVisible until the end of the transition.  With long transition time I could not make any glitches.